spec/unit/network/http/handler_spec.rb

```diff
@@ -95,9 +95,6 @@ def respond(text)
       expect(res[:content_type_header]).to eq("application/json")
       expect(res_body["issue_kind"]).to eq(Puppet::Network::HTTP::Issues::RUNTIME_ERROR.to_s)
       expect(res_body["message"]).to eq("Server Error: the sky is falling!")
-      expect(res_body["stacktrace"].is_a?(Array) && !res_body["stacktrace"].empty?).to be_truthy
```

Comment:
I verified the schema validates correctly (by adding back 'stacktrace' to the error object):

```
  1) Puppet::Network::HTTP::Error Puppet::Network::HTTP::Error::HTTPError should serialize to JSON that matches the error schema
     Failure/Error: JSON::Validator.validate!(@schema, json)

     JSON::Schema::ValidationError:
       The property '#/' contains additional properties ["stacktrace"] outside of the schema when none are allowed in schema c296f061-a030-5759-ab86-12b403e61ef0#
     # ./.bundle/ruby/2.3.0/gems/json-schema-2.1.1/lib/json-schema/validator.rb:62:in `validation_error'
     # ./.bundle/ruby/2.3.0/gems/json-schema-2.1.1/lib/json-schema/attributes/additionalproperties.rb:29:in `validate'
     # ./.bundle/ruby/2.3.0/gems/json-schema-2.1.1/lib/json-schema/validator.rb:95:in `block in validate'
     # ./.bundle/ruby/2.3.0/gems/json-schema-2.1.1/lib/json-schema/validator.rb:93:in `each'
     # ./.bundle/ruby/2.3.0/gems/json-schema-2.1.1/lib/json-schema/validator.rb:93:in `validate'
     # ./.bundle/ruby/2.3.0/gems/json-schema-2.1.1/lib/json-schema/schema.rb:38:in `validate'
     # ./.bundle/ruby/2.3.0/gems/json-schema-2.1.1/lib/json-schema/validator.rb:213:in `validate'
     # ./.bundle/ruby/2.3.0/gems/json-schema-2.1.1/lib/json-schema/validator.rb:374:in `validate!'
     # ./spec/lib/matchers/json.rb:116:in `matches?'
     # ./spec/unit/network/http/error_spec.rb:17:in `block (3 levels) in <top (required)>'

Finished in 8.24 seconds (files took 1.06 seconds to load)
```